pf/tests/testdata/basicprogram/Pulumi.yaml

```diff
@@ -171,11 +171,28 @@ outputs:
               - sshd
             port: 22
         protocol: ssh
-  tupleElementAttrRes__actual: ${tupleElementAttrRes.tuplesOptionals}
-  tupleElementAttrRes__expected:
-    - { t0: false, t1: "no" }
-    - { t0: true, t1: "yes"}
+
+  # TODO this is currently failing.
+  # tupleElementAttrRes__actual: ${tupleElementAttrRes.tuplesOptionals}
+  # tupleElementAttrRes__expect:
```

Comment:
__expected vs __expect was not being picked up, and this test fails, unfortunately @iwahbe

Reply:
Tracked at https://github.com/pulumi/pulumi-terraform-bridge/issues/1012